catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -153,10 +154,23 @@ private void updateIdentityNodeName(Metacard metacard)
                 extrinsicObjectType.setName(internationalStringTypeHelper.create(siteName)));
     Metacard identityMetacard = getRegistryMetacardFromRegistryPackage(registryPackage);
     if (identityMetacard != null) {
+      copyTransientAttributes(identityMetacard, metacard);
       federationAdminService.updateRegistryEntry(identityMetacard);
     }
   }
 
+  private void copyTransientAttributes(Metacard updateMetacard, Metacard existingMetacard) {
```

Comment:
I know this was taken from another location but was wondering if this might be cleaner ```RegistryObjectMetacardType.TRANSIENT_ATTRIBUTES
        .stream()
        .filter(
            key ->
                updateMetacard.getAttribute(key) == null
                    && existingMetacard.getAttribute(key) != null)
        .forEach(key -> updateMetacard.setAttribute(existingMetacard.getAttribute(key)));``` but now that I write it out I am not sure.

Reply:
I usually don't like streams but I really think this reads better 👍